var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -315,11 +315,17 @@ def write_library_dirs(f, dirs):
                     f.write("include_dirs = {0}\n".format(lapack_header_dirs))
                     f.write("extra_link_args = {0}\n".format(self.spec["lapack"].libs.ld_flags))
 
-            if "^armpl-gcc" in spec:
-                f.write("[blas]\n")
-                f.write("libraries = {0}\n".format(lapackblas_lib_names))
-                write_library_dirs(f, lapackblas_lib_dirs)
-                f.write("include_dirs = {0}\n".format(lapackblas_header_dirs))
+            if "^armpl-gcc" in spec or "^acfl" in spec:
+                if spec.satisfies("+blas"):
+                    f.write("[blas]\n")
+                    f.write("libraries = {0}\n".format(lapackblas_lib_names))
+                    write_library_dirs(f, lapackblas_lib_dirs)
+                    f.write("include_dirs = {0}\n".format(lapackblas_header_dirs))
+                if spec.satisfies("+lapack"):
+                    f.write("[lapack]\n")
+                    f.write("libraries = {0}\n".format(lapackblas_lib_names))
+                    write_library_dirs(f, lapackblas_lib_dirs)
+                    f.write("include_dirs = {0}\n".format(lapackblas_header_dirs))
```

Comment:
```suggestion
                if spec.satisfies("+blas"):
                    f.write("[blas]\n")
                    f.write("libraries = {0}\n".format(blas_lib_names))
                    write_library_dirs(f, blas_lib_dirs)
                    f.write("include_dirs = {0}\n".format(blas_header_dirs))
                if spec.satisfies("+lapack"):
                    f.write("[lapack]\n")
                    f.write("libraries = {0}\n".format(lapack_lib_names))
                    write_library_dirs(f, lapack_lib_dirs)
                    f.write("include_dirs = {0}\n".format(lapack_header_dirs))
```
I know there's only a single library for both blas and lapack, but I think it makes more visual sense to organize them like this.

Reply:
> I know there's only a single library for both blas and lapack

That's not always true: Netlib BLAS/LAPACK, BLIS/libFlame, and you can even build OpenBLAS without LAPACK.

Reply:
Yes, I'm only talking about armpl/acfl

Reply:
@adamjstewart Thank you for the suggestion !!